wallet/src/de/schildbach/wallet/ui/dashpay/ContactsFragment.kt

```diff
@@ -171,9 +176,7 @@ class ContactsFragment : BottomNavFragment(R.layout.fragment_contacts_root), Tex
             if (it.data != null && it.data.isNotEmpty()) {
                 showSuggestedUsers(it.data)
             } else {
-                if (contactsAdapter.results.isEmpty()) {
-                    showEmptyPane()
-                }
+                showSuggestedUsers(null)
```

Comment:
This will show the empty results with the search box still visible instead of covering entire fragment.